public/app/features/alerting/unified/rule-list/StateView.tsx

```diff
@@ -0,0 +1,153 @@
+import { css } from '@emotion/css';
+import { useMemo } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Counter, Pagination, Stack, useStyles2 } from '@grafana/ui';
+import { DEFAULT_PER_PAGE_PAGINATION } from 'app/core/constants';
+import { CombinedRule, CombinedRuleNamespace } from 'app/types/unified-alerting';
+import { PromAlertingRuleState } from 'app/types/unified-alerting-dto';
+
+import { usePagination } from '..//hooks/usePagination';
+import { calculateTotalInstances } from '../components/rule-viewer/RuleViewer';
+import { ListSection } from '../rule-list/components/ListSection';
+import { createViewLink } from '../utils/misc';
+import { hashRule } from '../utils/rule-id';
+import {
+  getRuleGroupLocationFromCombinedRule,
+  getRulePluginOrigin,
+  isAlertingRule,
+  isGrafanaRulerRule,
+} from '../utils/rules';
+
+import { AlertRuleListItem } from './components/AlertRuleListItem';
+import { ActionsLoader, RuleActionsButtons } from './components/RuleActionsButtons.V2';
+
+interface Props {
+  namespaces: CombinedRuleNamespace[];
+}
+
+type GroupedRules = Map<PromAlertingRuleState, CombinedRule[]>;
+
+export const StateView = ({ namespaces }: Props) => {
```

Comment:
In this PR we are introducing this new component, but is still not used , right?

Reply:
We're using this one – it's the state view tab :)

<img width="325" alt="image" src="https://github.com/user-attachments/assets/ca0d0224-c146-407a-85a1-3e30b0f8942a">

Reply:
yes, my problem is that as the filters are not working, I cannot test this view yet